.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,21 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
+      - run: codespell --skip="*/misspelling.json" || true  # --ignore-words-list=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=15 --max-line-length=120 --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install -r requirements.txt
+      - run: mypy --ignore-missing-imports . || true
+      - run: pytest . || true
```

Comment:
When looking at the sample GitHub actions, it seems as if running this step (and the subsequent one) failed. Could it be because on the repo we currently use Python's built-in `unittest` library for testing instead of `pytest`?

If you could get the testing action to work out of the box, amazing, otherwise, you might have to change this to run the `python -m unittest` commands, i.e.:

```
python -m unittest augly.tests.audio_tests.functional_unit_tests
```

for each specific unit test (not sure if there is a command for `unittest` that allows you to run all unit tests at once).

Reply:
Another thing: in order to be able to run *all* unittests, you will need to install `torchvision` to run the `image_tests/pytorch_test.py`. We can also omit that one.